### ISSUE TITLE
Question - Post error 204 when sending data to database

### ISSUE THREAD

**Issue by rmszc81** [AppMetrics/InfluxDB#31]
Hi,

when I'm trying to post data to influx through AppMetrics.InfluxDB, I'm getting a 204 error (no content). This is caused 'cause the precision parameter is not sent on URL anymore.

Would u lend me a hand please?

Thanks!

**Comment by alhardy** [comment]
I haven't had this issue myself. What version of influxdb are you seeing this issue? I don't think I set the precision param, will double check.

**Comment by rmszc81** [comment]
I've tested with v1.5 of influxDB (downloaded today).

Later I discovered that, for some reason, the parameter "Precision" was not present in the influxdb.conf file. I added the parameter as:

Precision = "s"

And the problem with App.Metrics/InfluxDB 204 was gone for good. But, if this directive isn't present anymore in the influxdb config file, wouldn't be interesting to add this guy again?

For example:

metrics.InfluxDB.Precision = Precision.Seconds;

Thanks for the quick answer!

**Comment by alhardy** [comment]
Interesting. 

AppMetrics by default sends timestamps in unix nanoseconds, which I believe is the influxdb default.

Would there be a big benefit making this configurable with AppMetrics?

**Comment by meron1122** [comment]
I have this same issue too

**Comment by tbolon** [comment]
Yes, it is an interesting optimization, as described [in the docs](https://docs.influxdata.com/influxdb/v1.7/write_protocols/line_protocol_reference/).

If you send datas which are expected to be precise only at the second (or minute) level you should use the least coarsest precision, as the data will be better compressed.
